### PR TITLE
Compilation tweaks

### PR DESCRIPTION
**Pull request by chanson5000**
I ran into some issues when following along with the instructions in the _README.MD_ and I thought these changes would help ease set-up.

Since all S3 bucket names have to be unique, errors occur while creating a stack with _make provision-base_ when bucket names are not changed from their defaults in the _parameters.json_ file. Adding instructions to modify this file may help prevent this error which also occurred for me.

In _put-types.js_, the secrets manager region was set to us-east-1 by default, however during configuration my secrets were stored in us-east-2 which caused errors for me. I thought adding a configuration file like the _secrets-config.json_ next to the _parameters.json_ file would make it easier to make this change rather than tracking it down in the _put-types.js_ file. I found attempting to combine _secrets-config_ into _parameters_ creates an error when using aws-cli.

Added error handing in _put-types.js_ to make it more clear when aws-cli is trying to use secrets manager keys from the wrong region.

Making these changes made provisioning and deploying easier with less troubleshooting for me.

### CODE REVIEW

**Comment by tbrunia** [comment]
Thank you for the PR, Cory! It’s so great to hear your feedback. Awesome suggestions. It’s hard to notice those details unless someone else tries to work through the instructions. I hope you found the example valuable.

I just saw this message. I’m on a walk this morning and will review and merge later this afternoon.

File: utils/types/put-types.js [hunk 1]
```diff
@@ -8,8 +8,13 @@ const prompt = require('prompt');
 const identityBroker = require('../../temp/src/backend/identityBroker');
 const dataAccessorBroker = require('../../temp/src/backend/dataAccessorBroker');
 
+const secretsConfig = require('../../aws/cloud-formation/secrets-config.json');
+
+console.log("Using AWS Secrets Manager in region: " + secretsConfig.SecretsRegion);
```
Comment (tbrunia) [inline utils/types/put-types.js]: Need to add...
```
// eslint-disable-next-line no-console
```
above this line.

**Comment by tbrunia** [comment]
@chanson5000, thanks you again for the updates. They look really good. Just one request to add some eslint escape statements. Please kindly add these and I'll merge your PR.

**Comment by chanson5000** [comment]
I believe I have added the requested eslint escape statements. Thanks!

**Pull request merged**